graphql_api/types/repository/repository.graphql

```diff
@@ -25,6 +36,7 @@ type Repository {
     last: Int
     before: String
   ): PullConnection @cost(complexity: 10, multipliers: ["first", "last"])
+  # TODO - does this refer to commits on main only?
```

Comment:
If anyone can answer this question, please let us know. What happens for other branches?

Reply:
It's just on default branch I'm pretty sure. You can get any commit by its SHA with `Repository { commit(id: SHA) {} }` per the API docs. idk how that maps onto this file, however lol.

Reply:
`Repository.commits` can be any set of commits depending on the filters used (see: `filters: CommitsSetFilters`)

In the filters it specifies:
```
type CommitsSetFilters {
  hideFailedCI: Boolean
  branchName: String
  pullId: Int
  search: String
  states: [CommitState!]
  coverageStatus: [CommitStatus!]
}
```

Reply:
Dangg that's good to know thanks for clarifying @JerrySentry